src/benchq/algorithms/lde_solver.py

```diff
@@ -194,7 +201,7 @@ def inverse_blockencoding(
 
     prep, prep_prime = control_prep(k=k, beta=beta)
     prep_prime_dag = prep_prime.inverse()
-    grid_qubits = ceil(np.log2(k)) + 1
+    grid_qubits = ceil(np.log2(k))
```

Comment:
Didn't there used to be a +1 on this register? What happened?

Reply:
Yes, it was! I think if there is +1, then the control qubit is outside of the PREP_int block